src/pages/signUp/Welcome.tsx

```diff
@@ -1,7 +1,6 @@
 import { Box, Stack, useTheme } from "@mui/material";
 import { FilledButton, Typography } from "elements";
 import { FunctionComponent, useState } from "react";
```

Comment:
If you are removing this page from the route, i am wondering if this was necessary? or just making sure we dont authenticate for sure